src/components/ElevationSlider.vue

```diff
@@ -10,9 +10,18 @@
       v-on:change="onInputChange"
       :hideLabel="true"
       lazy direction="btt" tooltip="always" tooltipPlacement="left" height="200px" ref="slider">
-      <template v-slot:tooltip="{ value }">
-        <div class="vue-slider-dot-tooltip-inner vue-slider-dot-tooltip-inner-left vue-slider-dot-tooltip-text">{{
-          Math.round(value) }} m MSL</div>
+      <template v-slot:tooltip>
+        <div class="vue-slider-dot-tooltip-inner vue-slider-dot-tooltip-inner-left vue-slider-dot-tooltip-text" >
+          <v-text-field ref="tooltipInput" v-if="isEditingTooltip"
+            @keydown.escape.stop="disableTooltipEdit"
+            @keydown.enter.stop="acceptTooltipEdit"
+            @blur="acceptTooltipEdit"
+            v-model.number="currentTooltipValue"
```

Comment:
Add hide-spin-buttons options?
```suggestion
            v-model.number="currentTooltipValue"
            hide-spin-buttons
```